slp_abacus/resources/abacus-example/abacus_merged.json

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
This file should be moved to the examples/abacus folder in the root of the repository, as it is not a resource itself of the SLP, but an example for this specific format. Maybe, it could be useful for testing purposes. In that, case we should also copy it to some folder under `slp_abacus/tests`.